libraries/core-react/src/components/Menu/MenuItem.tsx

```diff
@@ -124,7 +127,7 @@ export const MenuItem = memo(
     return (
       <ListItem
         {...props}
-        ref={useCombinedRefs<HTMLLIElement>(ref, (el) => {
+        ref={useCombinedRefs<HTMLButtonElement>(ref, (el) => {
           if (el !== null && isFocused) {
             el.focus()
           }
```

Comment:
Do we still need the `!disabled` check on line 137 now that its a `button`  element, the onClick handler should not be triggered when disabled.

Reply:
Good point, we did not need it